src/components/navigation/header.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from "react";
+import Menu from "./menu";
+import { userService } from "../../api/user";
+import ukraine_language from "../../assets/images/ukraine_language.png";
+import english_language from "../../assets/images/english_language.png";
+import white_theme from "../../assets/images/white_theme.png";
+import dark_theme from "../../assets/images/dark_theme.png";
+import burger_menu from "../../assets/images/burger_menu.png";
+
+export default function Header() {
+    const [language, setLanguage] = useState(ukraine_language);
+    const [theme, setTheme] = useState(white_theme);
+
+    const [isOpen, setIsOpen] = useState(false);
+    const [data, setData] = useState("Null");
+
+    const changeLanguage = () => {
+        if (language === ukraine_language) setLanguage(english_language);
+        else setLanguage(ukraine_language);
+    };
+
+    const changeTheme = () => {
+        if (theme === white_theme) setTheme(dark_theme);
+        else setTheme(white_theme);
+    };
+
+    useEffect(() => {
+        userService.getUser().then((res) => {
```

Comment:
I think it should be taken to the service, not called here. And be called when the main page is loaded